### PR TITLE
fix: #40

### PR DESCRIPTION
**Pull request by chenzesam**
1. when don't have modules property and return;

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -67,17 +67,19 @@ module.exports = (options) => {
         // Spit out stats during bundle generation
         generateBundle : (_, bundles) => {
             Object.values(bundles).forEach((bundle, idx) => {
+                if (!bundle.modules) return;
```
Comment (tivac) [inline index.js]:
```suggestion
                if (!bundle.modules) {
                    return;
                }
```

**Comment by tivac** [comment]
Thanks!

**Comment by tivac** [comment]
Released as `rollup-plugin-sizes@1.0.2`

**Comment by chenzesam** [comment]
@tivac Thank you for your quick response!

**Pull request merged**